momento-sdk/src/main/java/momento/sdk/config/TopicConfigurations.java

```diff
@@ -0,0 +1,120 @@
+package momento.sdk.config;
+
+import java.time.Duration;
+import momento.sdk.config.transport.GrpcConfiguration;
+import momento.sdk.config.transport.StaticTransportStrategy;
+import momento.sdk.config.transport.TransportStrategy;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/** Prebuilt {@link TopicConfiguration}s for different environments. */
+public class TopicConfigurations {
+  /**
+   * Provides defaults suitable for a medium-to-high-latency dev environment. Permissive timeouts,
+   * relaxed latency and throughput targets.
+   */
+  public static class Laptop extends TopicConfiguration {
+
+    private Laptop(TransportStrategy transportStrategy, Logger logger) {
+      super(transportStrategy, logger);
+    }
+
+    /**
+     * Provides the latest recommended configuration for a dev environment.
+     *
+     * <p>This configuration may change in future releases to take advantage of improvements we
+     * identify for default configurations.
+     *
+     * @return the latest Laptop configuration
+     */
+    public static TopicConfiguration latest() {
+      return Laptop.v1();
+    }
+
+    /**
+     * Provides v1 configuration for a laptop environment. This configuration is guaranteed not to
+     * change in future releases of the Momento Java SDK.
+     *
+     * @return the v1 laptop configuration
+     */
+    public static TopicConfiguration v1() {
```

Comment:
let's not include the v1s yet. We can just provide `latest` and add a v1 once some users have gotten some mileage on it.